dkg/src/dkg_manager/mod.rs

```diff
@@ -122,11 +130,12 @@ impl<DKG: DKGTrait, P: DKGPrivateParamsProvider<DKG>> DKGManager<DKG, P> {
         let (agg_trx_tx, mut agg_trx_rx) = aptos_channel::new(QueueStyle::KLAST, 1, None);
         self.agg_trx_tx = Some(agg_trx_tx);
 
+        let mut dkg_start_event_rx = dkg_start_event_rx.into_stream();
         let mut close_rx = close_rx.into_stream();
         while !self.stopped {
             let handling_result = tokio::select! {
                 dkg_start_event = dkg_start_event_rx.select_next_some() => {
-                    self.process_dkg_start_event(dkg_start_event).await
+                    self.process_dkg_start_event(dkg_start_event.ok()).await
```

Comment:
why this is Option?

Reply:
since now it's a oneshot, `dkg_start_event` is a result?

Reply:
hmm, we should shutdown if it's error?

Reply:
it's error only when epoch manager is dropped or epoch manager dropped the tx, both should be unreachable?